kustomize/commands/edit/fix/fix_test.go

```diff
@@ -130,19 +365,26 @@ labels:
     foo: bar
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-`)
-	fSys := filesys.MakeFsInMemory()
-	testutils_test.WriteTestKustomizationWith(fSys, kustomizationContentWithOutdatedCommonLabels)
-	cmd := NewCmdFix(fSys, os.Stdout)
-	assert.NoError(t, cmd.RunE(cmd, nil))
+`,
+		},
+	}
+	for _, test := range tests {
+		fSys := filesys.MakeFsInMemory()
+		testutils_test.WriteTestKustomizationWith(fSys, []byte(test.input))
+		for filename, content := range test.files {
+			require.NoError(t, fSys.WriteFile(filename, []byte(content)))
+		}
+		cmd := NewCmdFix(fSys, os.Stdout)
+		require.NoError(t, cmd.RunE(cmd, nil))
 
-	content, err := testutils_test.ReadTestKustomization(fSys)
-	assert.NoError(t, err)
-	assert.Contains(t, string(content), "apiVersion: ")
-	assert.Contains(t, string(content), "kind: Kustomization")
+		content, err := testutils_test.ReadTestKustomization(fSys)
+		require.NoError(t, err)
+		require.Contains(t, string(content), "apiVersion: ")
+		require.Contains(t, string(content), "kind: Kustomization")
 
-	if diff := cmp.Diff(expected, content); diff != "" {
-		t.Errorf("Mismatch (-expected, +actual):\n%s", diff)
+		if diff := cmp.Diff([]byte(test.expected), content); diff != "" {
+			t.Errorf("%s: Mismatch (-expected, +actual):\n%s", test.name, diff)
+		}
```

Comment:
Optional nit: We can replace these lines with `require.Empty(t, cmp.Diff([]byte(test.expected), content))` since `t.Run()` will print the sub-test name and `require.Empty()` will print `diff` if it isn't empty.

Reply:
done